tests/playwright/src/model/pages/pull-image-page.ts

```diff
@@ -39,7 +39,7 @@ export class PullImagePage extends BasePage {
     this.imageNameInput = page.getByLabel('Image to Pull');
   }
 
-  async pullImage(imageName: string, tag = '', timeout = 60000): Promise<ImagesPage> {
+  async pullImage(imageName: string, tag = '', timeout = 120000): Promise<ImagesPage> {
```

Comment:
I would change rather the place you call the `pullImage` from than changing it in everywhere. Look here: https://github.com/containers/podman-desktop/blob/main/tests/playwright/src/specs/volume-smoke.spec.ts#L122. And make sure to have enought timeout set for the test function in the spec.

Reply:
That makes complete sense. I've put it in the correct place and added 60 seconds to the timeout of the case, as that's what I added to the pullImage part.